docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1848,7 +1856,9 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0068W":"Failed to set proxy authorizations. Error=
+### ZWED0068W
+
+  Failed to set proxy authorizations. Error=
 
```

Comment:
Do we need Error=**%s**

Reply:
i havent touched these yet, i was just bulk fixing formatting.